magento/magento2
# logging-in in order to checkout when guest checkout is disabled #6015

https://github.com/magento/magento2/issues/6015





### Description (*)

Issue appear in case if "login_redirect" value in cookie missing. This case was reproduced with login using dialog opened by click "Proceed to Checkout" on cart page. 
### Fixed Issues (if relevant)

1. Fixes magento/magento2#6015: logging-in in order to checkout when guest checkout is disabled

### Manual testing scenarios (*)

1. Go to admin panel Stores -> (Settings) Configuration -> (Sales) Checkout and Set Allow Guest Checkout = No in Checkout Options. Save config.
2. Open application with clean cookie in browser (no login_redirect value should be defined)
3. Add product to cart without login
4. Go to cart page and click "Proceed to Checkout" on right side
5. Add login data in dialog and login

### Questions or comments


### Contribution checklist (*)
